lib/web_ui/lib/src/engine/platform_views/content_manager.dart

```diff
@@ -223,3 +240,18 @@ class PlatformViewManager {
     return result;
   }
 }
+
+DomElement _defaultFactory(
+  int viewId, {
+  Object? params,
+}) {
+  params!;
+  params as _DefaultFactoryParams;
```

Comment:
Can this be one-lineized as?

```suggestion
  params! as _DefaultFactoryParams;
```

Reply:
Good example with the `foo.toString()`, thanks!